test/tests/functional/pbs_eligible_time.py

```diff
@@ -148,3 +148,48 @@ def test_job_array(self):
                     raise PtlLogMatchError(rc=1, rv=False, msg=e.msg)
             else:
                 raise PtlLogMatchError(rc=1, rv=False, msg=e.msg)
+
+    def test_after_depend(self):
+        """
+        Make sure jobs accrue eligible time (or not) approprately with an
+        after dependency
+        """
+
+        self.server.manager(MGR_CMD_SET, NODE,
+                            {'resources_available.ncpus': 2},
+                            id=self.mom.shortname)
+        J1 = Job(TEST_USER)
+        jid1 = self.server.submit(J1)
+        attribs = {'job_state': 'R', 'accrue_type': 3}
```

Comment:
Can you please use a variable instead of using numbers. This will make it clear what kind of accrue type we are looking for.